app/src/main/java/com/developersbreach/composeactors/ComposeActorsApp.kt

```diff
@@ -6,7 +6,6 @@ import timber.log.Timber
 
 @HiltAndroidApp
 class ComposeActorsApp : Application() {
-
```

Comment:
The existing rules specified removes the line between class declaration and classes content, this is severe since it reduces readability.
To retain then we can use
```kotlin
ktlint_standard_no-empty-first-line-in-class-body = disabled
```